cypress/integration/visual-regression.spec.js

```diff
@@ -44,138 +29,38 @@ describe('Percy Snapshots', () => {
-      cy.visit('/#Basics/Buttons/Full%20Width')
-      cy.percySnapshot('Full-width button')
-    })
-
-    it('Creates a snapshot for toggle variation of a button', () => {
-      cy.visit('/#Basics/Buttons/Toggle')
-      cy.percySnapshot('Toggle button')
+      cy.visit('/#Basics/Buttons/United')
+      cy.percySnapshot('All buttons')
     })
   })
 
   describe('Alerts', () => {
     it('Creates a snapshot for primary variation of an alert', () => {
-      cy.visit('/#Basics/Alerts/Primary')
-      cy.percySnapshot('Primary alert')
-    })
-
-    it('Creates a snapshot for success variation of an alert', () => {
-      cy.visit('/#Basics/Alerts/Success')
-      cy.percySnapshot('Success alert')
-    })
-
-    it('Creates a snapshot for warning variation of an alert', () => {
-      cy.visit('/#Basics/Alerts/Warning')
-      cy.percySnapshot('Warning alert')
-    })
-
-    it('Creates a snapshot for danger variation of an alert', () => {
-      cy.visit('/#Basics/Alerts/Danger')
-      cy.percySnapshot('Danger alert')
+      cy.visit('/#Basics/Alerts/United')
+      cy.percySnapshot('All alerts')
     })
   })
 
   describe('Badges', () => {
     it('Creates a snapshot for gray variation of a badge', () => {
-      cy.visit('/#Basics/Badges/Badge%20gray')
-      cy.percySnapshot('Badge gray')
-    })
-
-    it('Creates a snapshot for primary variation of a badge', () => {
-      cy.visit('/#Basics/Badges/Badge%20primary')
-      cy.percySnapshot('Badge primary')
-    })
-
-    it('Creates a snapshot for warning variation of a badge', () => {
-      cy.visit('/#Basics/Badges/Badge%20warning')
-      cy.percySnapshot('Badge warning')
-    })
-
-    it('Creates a snapshot for danger variation of a badge', () => {
-      cy.visit('/#Basics/Badges/Badge%20danger')
-      cy.percySnapshot('Badge danger')
-    })
-
-    it('Creates a snapshot for success variation of a badge', () => {
-      cy.visit('/#Basics/Badges/Badge%20success')
-      cy.percySnapshot('Badge success')
+      cy.visit('/#Basics/Badges/United')
+      cy.percySnapshot('All badge')
     })
   })
 
   describe('Text fields', () => {
     it('Creates a snapshot for default text field', () => {
-      cy.visit('/#Basics/TextField/Default')
-      cy.percySnapshot('Default text field')
-    })
-
-    it('Creates a snapshot for username text field', () => {
-      cy.visit('/#Basics/TextField/Username')
-      cy.percySnapshot('Username text field')
-    })
-
-    it('Creates a snapshot for password text field', () => {
-      cy.visit('/#Basics/TextField/Password')
-      cy.percySnapshot('Password text field')
-    })
-
-    it('Creates a snapshot for full-width text field', () => {
-      cy.visit('/#Basics/TextField/Full%20Width')
-      cy.percySnapshot('Full-width text field')
+      cy.visit('/#Basics/TextField/All')
+      cy.percySnapshot('All text fields')
     })
   })
 
-  describe('Loading spinners', () => {
+  /*describe('Loading spinners', () => {
```

Comment:
Spinners don't seem to sync with Percy, always triggering the difference and unnecessarily delaying PRs.
But if we keep them we may notice if one day they disappear.
What do you guys think?

Reply:
supposedly percy pauses the animations to take the screenshot, there must be some delay causing that to not always happen at the same frame

Reply:
removing it should be okay